doc/legacy_mode/main.md

```diff
@@ -34,36 +30,30 @@ The extension supports configuration of the tools:
 * [Rustfmt Configuration](rustfmt_configuration.md)
 * [Rustsym Configuration](rustsym_configuration.md)
 
-This mode supports configuration via the configuration parameters.
+You may also configure Legacy Mode via configuration parameters.
 
 ## Configuration Parameters
 
 ### Show Output
 
-The `"rust.showOutput"` configuration parameter controls whether the output channel should be shown when [a cargo command is started executing](../cargo_command_execution.md).
+The `"rust.showOutput"` configuration parameter controls whether the Output channel should be shown when [a Cargo command starts executing](../cargo_command_execution.md). All Cargo command other than `build` and `check` trigger this event.

```

Comment:
Replace **all** with **any** or add **s** to **command**.
Actually the PR make different other than you are describing.
I am going to write what the PR does.